frontend/src/components/Profile/BasicTabs.tsx

```diff
@@ -85,6 +86,13 @@ export default function BasicTabs({
               {...a11yProps(3)}
             />
           )}
+          {(role === 'moderator' || role === 'admin') && (
```

Comment:
I think it should be in one block. You should wrapp this moderation tabs in fragment